tomls/omnibus-arbitrum-sepolia/perps/collaterals/btc.toml

```diff
@@ -41,7 +41,6 @@ args = [
         "<%= settings.dai_address %>",
         "<%= settings.usdc_address %>",
         "<%= settings.USDe_address %>",
-        "<%= settings.sUSDe_address %>",
```

Comment:
I'd expect `rewards_distributor_external_package` to change as well to use the latest version that introduced the change

Reply:
https://github.com/Synthetixio/synthetix-deployments/pull/347/files#diff-90e83d999478043debde0c008cf77d10cdfa92c4e3be7da8c40755ab8f7a4bc1R91

Reply:
could be a situation where having no collaterals deposited, leads to issues if hooked up to a distributor too... waiting for leo to be online to check it out.